resources/assets/uad_lists.json

```
@@ -5879,7 +5879,7 @@
   {
     "id": "com.samsung.android.app.motionpanoramaviewer",
     "list": "Oem",
-    "description": "Motion panorama viewer\nLet you see the result of a motion panorama\nhttps://www.samsung.com/global/galaxy/what-is/motion-panorama/\n",
+    "description": "Motion panorama viewer\nLet's you see the result of a motion panorama\nhttps://www.samsung.com/global/galaxy/what-is/motion-panorama/\n",
```

```suggestion
    "description": "Motion panorama viewer\nLets you see the result of a motion panorama\nhttps://www.samsung.com/global/galaxy/what-is/motion-panorama/\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.